app/views/users/users.html.slim

```diff
@@ -0,0 +1,10 @@
+- @users.each do |u|
+  .user data-id=u.id
+    = link_to user_path(id: u.id.to_i) do
```

Comment:
use can use as parameters whole object user_path(u)
btw, in case u.id.to_i to_i is redundant because id is integer